.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: ci
+name: ci NSolid
```

Comment:
```suggestion
name: ci
```

Reply:
I got the names mixed while creating the new files.
Fixed